app/importer.js

```diff
@@ -279,8 +281,12 @@ importer.on('import-success', (e) => {
       appActions.changeSetting(settings.SHOW_BOOKMARKS_TOOLBAR, true)
     }
   }
+  setImmediate(() => {
+    windowActions.setImportBrowserDataDetail()
```

Comment:
This is code is not called, because `windowActions` is not propagated to the windows. We could be using appState and introduce new object `ui` and fill this things into it. We could clear it at the end. This would be a first step of migrating window state ui into app state ui. @bsclifton wdyt?

Reply:
Good call- I think this will have to be AppState actually.  We can could even show badges or progress, similar to that which was added with https://github.com/brave/browser-laptop/pull/9864

Reply:
now we have appAction for import :smiley: